src/Expansion.php

```diff
@@ -332,6 +330,35 @@ private function lookupAstOptional(Ast $label, Ast $context) : Ast {
         return $result;
     }
 
+    private function compileExpanderCall(callable $function, Ast $expanderAst, $cg): callable {
+        $expander = new \ReflectionFunction($function);
+        $delayedArg = function(){};
+        if ($expander->getParameters()) {
+            if ($expander->getParameters()[0]->hasType() && Ast::class === $expander->getParameters()[0]->getType()->getName()) {
```

Comment:
Travis is failing because it seems PHP7 has no https://www.php.net/manual/en/class.reflectionnamedtype.php